### DISCUSSION TITLE
What would a port of Hunger Games to React Native entail ?

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/hunger-games#484] [Ideas]
What would a port of Hunger Games to React Native entail ?

- Assuming a single code base, and Android/iOS/Web versions ? 
- Could it be a Summer of Code project ? 
- Would we have the skills to mentor it ?

This was raised by Aman and Marvin during the Flutter call (we initially considered a Flutter version)

**Comment by alexfauquette** [comment]
I've no notion of React Native. I would be interested to follow the project but don't have the bandwidth to clean hunger game + learn react-native

**Comment by alexfauquette** [comment]
An additional point I had in mind this week-end: Why doing a port to react-native?

If it's to be able to install the app, it's already feasible. But not sure it makes sens.
If it's for integration into the main app, does React-Native helps? I'm not sure you can mix a react-native with flutter codes

**Reply by nobeeakon** [reply]
think would be better to have it as a PWA (I'm assuming Alexadre was pointing to that) than doing a port to react native
